superset/assets/src/SqlLab/components/ExploreResultsButton.jsx

```diff
@@ -159,11 +159,12 @@ class ExploreResultsButton extends React.PureComponent {
       </div>);
   }
   render() {
+    const allowsSubquery = this.props.database && this.props.database.allows_subquery;
```

Comment:
nit: deconstruct props on the line before then you can just use `allows_subquery` in `disabled` param
```
const { 
    database: {
           allows_subquery
} } = this.props
```
https://github.com/lyft/marketdx/blob/master/src/components/Diagnosis/index.jsx#L55